pkg/pulumiyaml/run_test.go

```diff
@@ -2656,7 +2652,10 @@ func TestResourceSecretObjectProperties(t *testing.T) {
 name: test-yaml
 runtime: yaml
 config:
-  props: {}
+  props:
+    type: object
+    default:
+      foo: "bar"
```

Comment:
What happens if you have default `foo: "xyz"` here and pass the config map below?  Does the default get overwritten correctly?